Sources/SwiftFrame/main.swift

```diff
@@ -18,7 +18,7 @@ struct SwiftFrame: ParsableCommand {
     static let configuration = CommandConfiguration(
         commandName: "swiftframe",
         abstract: "CLI application for speedy screenshot framing",
-        version: "2.2.0",
+        version: "2.3.0",
```

Comment:
The `textAlignment` change in the config seems backwards incompatible. Following semantic versioning, I think we need to bump the major version number instead, therefore.